docs/crocks/Star.md

```diff
@@ -0,0 +1,61 @@
+# Star
+
+`Monad m => Star a (m b)`
```

Comment:
Let's keep :corn: and only try to say `Star a b`, but here are some ideas I came up with:

- `Star a (Monad b)`

- `Star a b`
   ` . Monad m => b ~> m b`

- `Star a (m b)`
    ` . Monad m`

The superstar inside me says "keep the laws implicit until skeleton is drawn!"